acceptance-test-wiki.md

```diff
@@ -0,0 +1,84 @@
+
+3) The functionality of the top-level tests for each user-type is defined in the `user-utilities` directory. For ex. the blog editor tests are written within the `spec/blog-editor-tests` directory, and the functionality of the tests are defined in the `user-utilities/blog-post-editor-utils.js` file.
+> Note: A utility file is maintained for each user type. The purpose of maintaining this file is to add methods specific to that user on top of the already provided basic methods. This file maintains a user class which is extended from the base class of puppeteer-utils.js . For ex. blog-post-admin-utils.js have a class e2eBlogPostAdmin which have methods like `createDraftBlogPostWithTitle`, `deleteDraftBlogPostWithTitle` etc. specific to Blog Admin only.
+
+
+4) The utility files are imported in the top-level test files and the methods are called to perform the required actions. For ex. in the `create-draft-and-delete-draft-blog-post.spec.js` file, the `createDraftBlogPostWithTitle` method is called to create a draft blog post with a given title. The `deleteDraftBlogPostWithTitle` method is called to delete the draft blog post with the given title.
+
+5) For each test, the user is created using the `userFactory` class. For ex. in the `create-draft-and-delete-draft-blog-post.spec.js` file, the `createNewBlogPostEditor` method is called to create a new blog post editor user. The `createNewBlogPostEditor` method is defined in the `user-factory.js` file. The `createNewBlogPostEditor` method creates a new user with the given username and returns the user object. The user object is used to perform the required actions (that are defined in the `user-utilities/*-utils.js`).
+
+6) After successful completition of any test step or any expectation, the `showMessage` method is called to log the progress. For ex. in the `create-draft-and-delete-draft-blog-post.spec.js` file, the `showMessage` method is called to log the progress after the draft blog post is created. The `showMessage` method is defined in the `show-message-utils.js` file.
+
+7) If there is any error during the test, then we throw errors in the expectation step or there would be timeout error if some component does not behave as intended.
+
+8) The `puppeteer-testing-utilities` directory contains all the utility files and helper functions, which you would require to write new acceptance tests. This directory can also be used to append more utility functions as when required or needed by the user.
+
+
+## Reference Links
+Blog Admin and Blog Editor Tests - 
+  [Blog Admin top-level tests](insert link when @AshishPatwal PR is merged)
```

Comment:
also, I haven't inserted link here. it can be inserted once @ashish-patwal's PR is merged